libs/common/src/vault/services/fido2/fido2-client.service.ts

```diff
@@ -110,9 +110,18 @@ export class Fido2ClientService implements Fido2ClientServiceAbstraction {
     let credTypesAndPubKeyAlgs: PublicKeyCredentialParam[];
     if (params.pubKeyCredParams?.length > 0) {
       // Filter out all unsupported algorithms
-      credTypesAndPubKeyAlgs = params.pubKeyCredParams.filter(
-        (kp) => kp.alg === -7 && kp.type === "public-key",
-      );
+      credTypesAndPubKeyAlgs = params.pubKeyCredParams.filter((kp) => {
+        if (typeof kp.alg == "string") {
```

Comment:
issue: we usually use strict equality wherever possible

```suggestion
        if (typeof kp.alg === "string") {
```